ibc-clients/ics08-wasm/types/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+name = "ibc-client-wasm-types"
+version      = { workspace = true }
+authors      = { workspace = true }
+license      = { workspace = true }
+rust-version = { workspace = true }
+edition      = { workspace = true }
+
+[dependencies]
+# external dependencies
+base64          = { version = "0.21", default-features = false, features = ["alloc"] }
+bytes           = { version = "1.2.1", default-features = false }
```

Comment:
I think that `bytes` dependency is not needed.

Reply:
[7baa0ba](https://github.com/cosmos/ibc-rs/pull/1031/commits/7baa0bac8a3fe744091cdb07a5daf89e33326e89)